### ISSUE TITLE
New plugin: hyphenate

### ISSUE THREAD

**Issue by Deraen** [hashobject/perun#33]
Depending on which phase this is done, might require splitting rending task (#30) or something. Best candidates are probably to hyphenate hiccup data or html.

I'm working on converting python hyphenation implementation to Clojure and once that is done it should be quite easy to implement a task for this.

**Comment by podviaznikov** [comment]
@Deraen I saw that you are using hyphenate on your blog. Can you make separate task/plugin or you are blocked on something?

**Comment by Deraen** [comment]
I wonder if I should just release as a separate package as there is nothing Perun specific about hyphenation html files?

**Comment by podviaznikov** [comment]
I think separate package is good idea

**Comment by Deraen** [comment]
Done. Added mention to README.

**Comment by podviaznikov** [comment]
Thanks! Going to try it on my blog
